### PR TITLE
Feature/results screen UI

### PR DESCRIPTION
**Pull request by ach206**
- result screen
- refactored asset distribution and consumption
- added new asset, insufficient icon

### CODE REVIEW

File: example/src/screens/ResultsScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { View, Image, StyleSheet, SafeAreaView, Text } from 'react-native';
+import { Spacing, Sizing, Colors, Typography } from '../styles';
+import * as Assets from '../../../src/assets';
```
Comment (Haplo-Dragon) [inline example/src/screens/ResultsScreen.tsx:4]: If we exported the assets from `src/index.ts`, this import might get cleaner? But I don't know if that's something we wanna do (expose the assets as an importable part of the SDK). Either way, it's very minor.

**Review by Haplo-Dragon — APPROVED** [review]
Looks good to me! Refactoring the assets into their own file was a great change.

**Review by bobbys-dev — APPROVED** [review]

File: example/src/screens/ResultsScreen.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ach206) [reply example/src/screens/ResultsScreen.tsx:4]: Let me see if we have the green light to expose them for consumption now. Initially we held off..but I think it should be ok to do it now tbh

Comment (ach206) [reply example/src/screens/ResultsScreen.tsx:4]: I agree, would def be cleaner. We should be ok to expose them. Let me double check

**Pull request merged**